jetty9-compat-base/src/test/java/com/google/apphosting/vmruntime/jetty9/LoggingIT.java

```diff
@@ -71,12 +74,12 @@ public void testGet() throws Exception {
       data = new Gson().fromJson(line, JsonData.class);
       assertThat(data.severity, equalTo("ERROR"));
       assertThat(
-          data.message, org.hamcrest.Matchers.containsString("LoggingServlet doGet: not null"));
+          data.message, org.hamcrest.Matchers.containsString("com.foo.bar: not null"));
```

Comment:
The old jetty9-compat logging didn't log with the logger name, it used "<calling class> <calling method name>: <message>"  (I know, its strange, but that's what they did)